Gems/EMotionFX/Code/EMotionFX/Pipeline/SceneAPIExt/Rules/MotionMetaDataRule.h

```diff
@@ -37,7 +37,7 @@ namespace EMotionFX::Pipeline::Rule
         AZStd::unique_ptr<EMotionFX::MotionEventTable> m_motionEventTable;
     };
 
-    class MotionMetaDataRule
+    class MotionMetaDataRule final
         : public ExternalToolRule<AZStd::shared_ptr<MotionMetaData>>
     {
     public:
```

Comment:
The `final` keyword on the destructor can now be removed.

Reply:
Fixed.